katheryne/genshin.py

```diff
@@ -0,0 +1,7 @@
+HOYOLAB_CHECKIN_URL = "https://webstatic-sea.mihoyo.com/ys/event/signin-sea/index.html?act_id=e202102251931481&lang=en-us"
+
+WEB_EVENT_NAME = "A Wanmin Welcome"
+WEB_EVENT_URL = "https://webstatic-sea.mihoyo.com/ys/event/e20210316cooking-sea/index.html?lang=en-us" 
+WEB_END_TIME = 1616687940
+
+NEXT_MAINTENANCE_TIME = 1615932000
```

Comment:
protip: unix timestamps are so 1980s, use `datetime(year, month, day, hour, minute, second, tz=...)` for actual human readable code. as a plus, it's timezone aware!